exasol_script_languages_container_tool/lib/utils/language_def_parser.py

```diff
@@ -0,0 +1,66 @@
+from typing import List, Tuple, Union
+from urllib.parse import urlparse
+
+from exasol_script_languages_container_tool.lib.api.get_language_activation_builder import (
+    BuiltInLanguageDefinitionURL,
+    LanguageDefinitionURL,
+    SLCLanguage,
+)
+from exasol_script_languages_container_tool.lib.api.language_activation import (
+    SLCParameter,
+)
+
+
+def parse_language_definition(
+    lang_def: str,
+) -> Tuple[str, Union[LanguageDefinitionURL, BuiltInLanguageDefinitionURL]]:
+    alias_end = lang_def.find("=")
+    alias = lang_def[0:alias_end]
+    url = lang_def[alias_end + 1 :]
+    if url.startswith("builtin"):
+        lang = url.replace("builtin_", "")
+        for slc_builtin_language in SLCLanguage:
```

Comment:
I think, you can use an enum like a map

Reply:
The thing here is the case-insenstive comparison. The Enums have names like "Python"....while the builtin name is lower case only

Reply:
Ah, your values are lower case and I think in also works on the values